skins/laika/src/view_models/Contact.ts

```diff
@@ -0,0 +1,17 @@
+import { Validity } from '@/view_models/Validity';
+
+export interface AddressValidity {
+    [key: string]: boolean
+}
+
+export interface InputField {
+    name: string,
+    value: string | null,
```

Comment:
when is `value` null? According to the data object it's initially an empty string.

Reply:
The drop down in its default state is `null` (empty string and undefined cause it to bug out) so I added `null` here. If we're putting on our fancy pants we can also create a new interface for just the subject dropdown but I figured this was okay and did not want to complicate things.